wikitemplate.md

```diff
@@ -228,3 +220,18 @@ Do not forget to make a backup of your entire `~/Library/Application\ Support/Br
 
 - [ ] Test that updating using `BRAVE_UPDATE_VERSION=0.8.3` env variable works correctly.
 - [ ] Test that using `BRAVE_ENABLE_PREVIEW_UPDATES=TRUE` env variable works and prompts for preview build updates.
+
+# CR Upgrade specific tests
```

Comment:
I don't think we need `CR specific tests`, we should probably be going through these cases every time we go through a pass. Performance wise, we should be making sure each release hasn't regressed in terms of performance. With the content test, it's easy enough to verify that we could probably add it into the general pass and not place it under `CR specific tests`. Thoughts @srirambv & @LaurenWags?